app/src/main/java/ch/epfl/sweng/eventmanager/notifications/NotificationPublisher.java

```diff
@@ -0,0 +1,25 @@
+package ch.epfl.sweng.eventmanager.notifications;
+
+import android.app.Notification;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.app.NotificationManagerCompat;
+import android.util.Log;
+
+public class NotificationPublisher extends BroadcastReceiver {
+
+    public static String NOTIFICATION_ID = "notification-id";
+    public static String NOTIFICATION = "notification";
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+
+        NotificationManagerCompat notificationManager =  NotificationManagerCompat.from(context);
+
+        Notification notification = intent.getParcelableExtra(NOTIFICATION);
+        int id = intent.getIntExtra(NOTIFICATION_ID, 0);
```

Comment:
It's a default value in case no id is received.

Reply:
Yes but default value could be anything, this is ok for me but in the future if you think it may change, it would be easier, otherwise leave it as it is.